tests/nodes/tests_Literal.cpp

```diff
@@ -14,7 +14,8 @@ TEST_CASE("Literal - Check for only lexical form") {
     CHECK(lit1.is_literal());
     CHECK(not lit1.is_variable());
     CHECK(not lit1.is_iri());
-    CHECK(lit1.lexical_form() == "Bugs Bunny");
+    lit1.lexical_form() == "Bugs Bunny";
```

Comment:
I think that is a left over from the PG tests. Please remove Bugs Bunny and any other similar names from the tests.

Reply:
Not sure I understand what you mean. I see the error and fixed that but idk what that has to do with the name

Reply:
I would like to have the mention of the Warner compic character "Bugs Bunny" removed and replaced by sth neutral